calico-vpp-agent/policy/policy_server.go

```diff
@@ -165,9 +165,11 @@ func NewPolicyServer(vpp *vpplink.VppLink, log *logrus.Entry) (*Server, error) {
 		return nil, errors.Wrapf(err, "Could not delete socket %s", config.FelixDataplaneSocket)
 	}
 
-	err = InstallFelixPlugin()
-	if err != nil {
-		return nil, errors.Wrap(err, "could not install felix plugin")
+	if withInstallPlugin {
```

Comment:
Not a fan of boolean flags passed to constructors.
We can maybe move the call to `InstallFelixPlugin()` up and call it right after `NewPolicyServer()`